apps/ledger-live-desktop/src/renderer/components/PostOnboardingHub/PostOnboardingHubContent.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useCallback } from "react";
+import { Flex, Button, Box, Text } from "@ledgerhq/react-ui";
+import { useHistory } from "react-router-dom";
+import { useDispatch } from "react-redux";
+import { useTranslation } from "react-i18next";
+import { usePostOnboardingHubState } from "@ledgerhq/live-common/postOnboarding/hooks/index";
+import { clearPostOnboardingLastActionCompleted } from "@ledgerhq/live-common/postOnboarding/actions";
+
+import PostOnboardingHub from ".";
+import { setDrawer } from "~/renderer/drawers/Provider";
+import { withV3StyleProvider } from "~/renderer/styles/StyleProviderV3";
+
+const PostOnboardingHubContent = () => {
+  const dispatch = useDispatch();
+  const { t } = useTranslation();
+  const history = useHistory();
+  const { lastActionCompleted, actionsState } = usePostOnboardingHubState();
+  const { actionCompletedHubTitle, actionCompletedPopupLabel } = lastActionCompleted ?? {};
+  const clearLastActionCompleted = useCallback(() => {
```

Comment:
This is not used right?
The idea is to call this function when this Component is unmounted so that next time the screen is opened, the title from the last action is not appearing.
You can do this by using the cleanup function in a `useEffect(() => clearLastActionCompleted, []);`